src-built-in/components/toolbar/stores/searchStore.js

```diff
@@ -26,6 +26,7 @@ var menuBlurHandler = Function.prototype;
 
 function mouseInWindow(win, cb) {
 	win.getBounds(function (err, bounds) {
+		if (err) FSBL.Clients.Logger.error('mouseInWindow->getBounds, error:', err);
```

Comment:
Looks like `FSBL.Clients.Logger.error` is repeated many many times, would be better if you just create a reference at the top of the file and just use it.

`const logger = FSBL.Clients.Logger`